.github/workflows/k8s-tests.yaml

```diff
@@ -0,0 +1,65 @@
+name: Execute tests on kubernetes
+on:
+  workflow_dispatch:
+    inputs:
+      test_file:
+        description: "Test file to run (e.g., constant_load.test.yaml)"
+        required: true
+        type: string
+        default: "constant_load.test.yaml"
+      k8s_namespace:
+        description: "Kubernetes namespace to run the test in"
+        required: true
+        type: string
+        default: "rudder-load"
+
+jobs:
+  run-k8s-test:
+    name: Run Kubernetes Test
+    runs-on: ubuntu-latest
```

Comment:
can we use our own github runners which are pre configured with all the necessary tooling pre installed and having the right access to k8s to run the tests.

Reply:
We can start with Github runners and update to our custom runner later as well.